deployments/warp_routes/DEUSD/ethereum-sei-config.yaml

```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=../schema.json
+tokens:
```

Comment:
We'll want a logo image for this. Do you have one handy or should I go find one?

Reply:
added